capplets/windows/Makefile.am

```diff
@@ -29,7 +29,6 @@ endif
 AM_CPPFLAGS =				\
 	$(WARN_CFLAGS)			\
 	$(MATECC_CAPPLETS_CFLAGS)	\
-	-DUIDIR=\""$(uidir)"\"		\
 	-DPIXMAPDIR=\""$(pixmapdir)"\"
```

Comment:
Can they be removed together ```-DPIXMAPDIR=\""$(pixmapdir)"\"```

Reply:
done

Test: removing `-DUIDIR=\"""\" -DPIXMAPDIR=\"""\"` from build log
```
$ ./autogen.sh --prefix=/usr && make V=1 &> make.log && grep PIXMAPDIR make.log
```